README.md

```diff
@@ -1,70 +1,84 @@
-# Getting Started with Create React App
+# Continents facts
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+> Continents facts is a a React and Redux SPA application that dispalys interesting information about various countries in the world.
+> It gets all the information that it displays using API request to a server.
 
-## Available Scripts
 
-In the project directory, you can run:
 
-### `npm start`
+## Built With
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- React
+- Redux
+- CSS
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Dev Dependencies
 
-### `npm test`
+- React-routerDOM
+- Axios
+- Redux-toolkit
+- React-test-renderer
+- createAsyncThunk
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Live Demo
 
-### `npm run build`
+- live demo: [Click here](https://tubular-cupcake-f62e8d.netlify.app)
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+### Video Link
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+- [Click here](https://drive.google.com/file/d/1VvIjeALtPvCYoudo-fFKRO6J01mEHtho/view?usp=sharing)
```

Comment:
- [ ] Your demo video is great but please make sure that your presentation video contains a highlight about a piece of the code the student finds interesting or is proud of making. :smile: 

- [ ] I noticed that your readme doesn't have a link to issue , please add it , that will help other contribute to your project and will make your readme more professional :smile:

Reply:
Hello @codecaiine  , I have already demonstrated an extra feature in my video which is the scroll-up icon I created to automatically scroll up to the top of the page.